src/Controller/SearchController.php

```diff
@@ -92,12 +98,25 @@ public function searchAction(Request $request): Response
             }
         }
 
+        //Prepare pager
+        $pager = $this->preparePager($page, $searchResults['total']);
+
+        //Prepare page help with
+        $pagingHelp = [
+            'start' => ($page - 1) * self::MAX_DISPLAYED_ITEMS_PER_PAGE + 1,
```

Comment:
Why do you apply a `+ 1`?
I may be wrong but I think that by doing this you're not getting the first result.